histolab/filters/compositions.py

```diff
@@ -139,3 +143,23 @@ def tissue_mask_filters(self) -> imf.Compose:
                 mof.BinaryFillHoles(structure=np.ones((20, 20))),
             ]
         )
+
+
+class _CustomFiltersComposition(FiltersComposition):
+    def __init__(self, *custom_filters) -> None:
+        super().__init__()
+        self.custom_filters = custom_filters
+
+    def set_custom_filters(self, *custom_filters):
```

Comment:
Please could you add the missing return type and the type for the `custom_filters`